<!DOCTYPE html> <html lang=""> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Nikita Rozanov</title> <meta name="author" content="Nikita Rozanov"> <meta name="description" content="Personal website of Nikita Rozanov. "> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700%7CRoboto+Slab:100,300,400,500,700%7CMaterial+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Nikita </span>Rozanov</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/artwork/">artwork</a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h1>Peer-Reviewed Publications</h1> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Tang2022" class="col-sm-8"> <div class="title">Rational design for high bioorthogonal fluorogenicity of tetrazine-encoded green fluorescent proteins</div> <div class="author"> Longteng Tang, Riley M. Bednar, <em>Nikita D. Rozanov</em>, Marcus L. Hemshorn, Ryan A. Mehl, and <a href="https://chemistry.oregonstate.edu/Fang" rel="external nofollow noopener" target="_blank">Chong Fang</a> </div> <div class="periodical"> <em>Natural Sciences</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.dx.doi.org/10.1002/ntls.20220028" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="external nofollow noopener">Link</a> </div> <div class="abstract hidden"> <p>The development of bioorthogonal fluorogenic probes constitutes a vital force to advance life sciences. Tetrazine-encoded green fluorescent proteins (GFPs) show high bioorthogonal reaction rate and genetic encodability but suffer from low fluorogenicity. Here, we unveil the real-time fluorescence mechanisms by investigating two site-specific tetrazine-modified superfolder GFPs via ultrafast spectroscopy and theoretical calculations. Förster resonance energy transfer is quantitatively modeled and revealed to govern the fluorescence quenching; for GFP150-Tet with a fluorescence turn-on ratio of ∼9, it contains trimodal subpopulations with good (P1), random (P2), and poor (P3) alignments between the transition dipole moments of protein chromophore (donor) and tetrazine tag (Tet-v2.0, acceptor). By rationally designing a more free/tight environment, we created new mutants Y200A/S202Y to introduce more P2/P1 populations and improve the turn-on ratios to ∼14/31, making the fluorogenicity of GFP150-Tet-S202Y the highest among all up-to-date tetrazine-encoded GFPs. In live eukaryotic cells, the GFP150-Tet-v3.0-S202Y mutant demonstrates notably increased fluorogenicity, substantiating our generalizable design strategy.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Tang2021" class="col-sm-8"> <div class="title">Switching between ultrafast pathways enables a green-red emission ratiometric fluorescent-protein-based Ca<sup>2+</sup> biosensor</div> <div class="author"> Longteng Tang, Shuce Zhang, Yufeng Zhao, <em>Nikita D. Rozanov</em>, Liangdong Zhu, Jiahui Wu, Robert E. Campbell, and <a href="https://chemistry.oregonstate.edu/Fang" rel="external nofollow noopener" target="_blank">Chong Fang</a> </div> <div class="periodical"> <em>International Journal of Molecular Sciences</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.dx.doi.org/10.3390/ijms22010445" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="external nofollow noopener">Link</a> </div> <div class="abstract hidden"> <p>Ratiometric indicators with long emission wavelengths are highly preferred in modern bioimaging and life sciences. Herein, we elucidated the working mechanism of a standalone red fluorescent protein (FP)-based Ca<sup>2+</sup> biosensor, REX-GECO1, using a series of spectroscopic and computational methods. Upon 480 nm photoexcitation, the Ca<sup>2+</sup>-free biosensor chromophore becomes trapped in an excited dark state. Binding with Ca<sup>2+</sup> switches the route to ultrafast excited-state proton transfer through a short hydrogen bond to an adjacent Glu80 residue, which is key for the biosensor’s functionality. Inspired by the 2D-fluorescence map, REX-GECO1 for Ca<sup>2+</sup> imaging in the ionomycin-treated human HeLa cells was achieved for the first time with a red/green emission ratio change (∆R/R<sub>0</sub>) of  300%, outperforming many FRET-and single FP-based indicators. These spectroscopy-driven discoveries enable targeted design for the next-generation biosensors with larger dynamic range and longer emission wavelengths.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Oscar2020" class="col-sm-8"> <div class="title">Dissecting optical response and molecular structure of fluorescent proteins with non-canonical chromophores</div> <div class="author"> Breland G. Oscar, Liangdong Zhu, Hayati Wolfendeen, <em>Nikita D. Rozanov</em>, Alvin Chang, Kenneth T. Stout, Jason W. Sandwisch, Joseph J. Porter, Ryan A. Mehl, and <a href="https://chemistry.oregonstate.edu/Fang" rel="external nofollow noopener" target="_blank">Chong Fang</a> </div> <div class="periodical"> <em>Frontiers in Molecular Biosciences</em> 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.dx.doi.org/10.3389/fmolb.2020.00131" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="external nofollow noopener">Link</a> </div> <div class="abstract hidden"> <p>Tracking the structural dynamics of fluorescent protein chromophores holds the key to unlocking the fluorescence mechanisms in real time and enabling rational design principles of these powerful and versatile bioimaging probes. By combining recent chemical biology and ultrafast spectroscopy advances, we prepared the superfolder green fluorescent protein (sfGFP) and its non-canonical amino acid (ncAA) derivatives with a single chlorine, bromine, and nitro substituent at the ortho site to the phenolate oxygen of the embedded chromophore, and characterized them using an integrated toolset of femtosecond transient absorption and tunable femtosecond stimulated Raman spectroscopy (FSRS), aided by quantum calculations of the vibrational normal modes. A dominant vibrational cooling time constant of  4 and 11 ps is revealed in Cl-GFP and Br-GFP, respectively, facilitating a  30 and 12% increase of the fluorescent quantum yield vs. the parent sfGFP. Similar time constants were also retrieved from the transient absorption spectra, substantiating the correlated electronic and vibrational motions on the intrinsic molecular timescales. Key carbon-halogen stretching motions coupled with phenolate ring motions of the deprotonated chromophores at ca. 908 and 890 cm<sup>−1</sup> in Cl-GFP and Br-GFP exhibit enhanced activities in the electronic excited state and blue-shift during a distinct vibrational cooling process on the ps timescale. The retrieved structural dynamics change due to targeted site-specific halogenation of the chromophore thus provides an effective means to design new GFP derivatives and enrich the bioimaging probe toolset for life and medical sciences.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Daly2020" class="col-sm-8"> <div class="title">Surface coating structure and its interaction with cytochrome c in EG6-coated nanoparticles varies with surface curvature</div> <div class="author"> Clyde A. Daly, Caley Allen, <em>Nikita D. Rozanov</em>, Gene Chong, Eric S. Melby, Thomas R. Kuech, Samuel E. Lohse, Catherine J. Murphy, Joel A. Pedersen, and <a href="https://chemistry.jhu.edu/directory/rigoberto-hernandez/" rel="external nofollow noopener" target="_blank">Rigoberto Hernandez</a> </div> <div class="periodical"> <em>Langmuir</em> 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.dx.doi.org/10.1021/acs.langmuir.0c00681" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="external nofollow noopener">Link</a> </div> <div class="abstract hidden"> <p>The composition, orientation, and conformation of proteins in biomolecular coronas acquired by nanoparticles in biological media contribute to how they are identified by a cell. While numerous studies have investigated protein composition in biomolecular coronas, relatively little detail is known about how the nanoparticle surface influences the orientation and conformation of the proteins associated with them. We previously showed that the peripheral membrane protein cytochrome c adopts preferred poses relative to negatively charged 3-mercaptopropionic acid (MPA)-gold nanoparticles (AuNPs). Here, we employ molecular dynamics simulations and complementary experiments to establish that cytochrome c also assumes preferred poses upon association with nanoparticles functionalized with an uncharged ligand, specifically ω-(1-mercaptounde-11-cyl)hexa(ethylene glycol) (EG6). We find that the display of the EG6 ligands is sensitive to the curvature of the surface - and, consequently, the effective diameter of the nearly spherical nanoparticle core - which in turn affects the preferred poses of cytochrome c.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Tollefson2019" class="col-sm-8"> <div class="title">Preferential binding of cytochrome c to anionic ligand-coated gold nanoparticles: A complementary computational and experimental approach</div> <div class="author"> Emily J. Tollefson, Caley R. Allen, Gene Chong, Xi Zhang, <em>Nikita D. Rozanov</em>, Anthony Bautista, Jennifer J. Cerda, Joel A. Pedersen, Catherine J. Murphy, Erin E. Carlson, and <a href="https://chemistry.jhu.edu/directory/rigoberto-hernandez/" rel="external nofollow noopener" target="_blank">Rigoberto Hernandez</a> </div> <div class="periodical"> <em>ACS Nano</em> 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.dx.doi.org/10.1021/acsnano.9b01622" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="external nofollow noopener">Link</a> </div> <div class="abstract hidden"> <p>Membrane-bound proteins can play a role in the binding of anionic gold nanoparticles (AuNPs) to model bilayers; however, the mechanism for this binding remains unresolved. In this work, we determine the relative orientation of the peripheral membrane protein cytochrome c in binding to a mercaptopropionic acid-functionalized AuNP (MPA-AuNP). As this is nonrigid binding, traditional methods involving crystallographic or rigid molecular docking techniques are ineffective at resolving the question. Instead, we have implemented a computational assay technique using a cross-correlation of a small ensemble of 200 ns long molecular dynamics trajectories to identify a preferred nonrigid binding orientation or pose of cytochrome c on MPA-AuNPs. We have also employed a mass spectrometry-based footprinting method that enables the characterization of the stable protein corona that forms at long time-scales in solution but remains in a dynamic state. Through the combination of these computational and experimental primary results, we have established a consensus result establishing the identity of the exposed regions of cytochrome c in proximity to MPA-AuNPs and its complementary pose(s) with amino-acid specificity. Moreover, the tandem use of the two methods can be applied broadly to determine the accessibility of membrane-binding sites for peripheral membrane proteins upon adsorption to AuNPs or to determine the exposed amino-acid residues of the hard corona that drive the acquisition of dynamic soft coronas. We anticipate that the combined use of simulation and experimental methods to characterize biomolecule-nanoparticle interactions, as demonstrated here, will become increasingly necessary as the complexity of such target systems grows.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Taylor2019" class="col-sm-8"> <div class="title">Delayed vibrational modulation of the solvated GFP chromophore into a conical intersection</div> <div class="author"> Miles A. Taylor, Liangdong Zhu, <em>Nikita D. Rozanov</em>, Kenneth T. Stout, Cheng Chen, and <a href="https://chemistry.oregonstate.edu/Fang" rel="external nofollow noopener" target="_blank">Chong Fang</a> </div> <div class="periodical"> <em>Physical Chemistry Chemical Physics</em> 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.dx.doi.org/10.1039/c9cp01077g" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="external nofollow noopener">Link</a> </div> <div class="abstract hidden"> <p>Green fluorescent protein (GFP) has revolutionized bioimaging and life sciences. Its successes have inspired modification of the chromophore structure and environment to tune emission properties, but outside the protein cage, the chromophore is essentially non-fluorescent. In this study, we employ the tunable femtosecond stimulated Raman spectroscopy (FSRS) and transient absorption (TA) to map the energy dissipation pathways of GFP model chromophore (HBDI) in basic aqueous solution. Strategic tuning of the Raman pump to 550 nm exploits the stimulated emission band to enhance excited state vibrational motions as HBDI navigates the non-equilibrium potential energy landscape to pass through a conical intersection. The time-resolved FSRS uncovers prominent anharmonic couplings between a global out-of-plane bending mode of ∼227 cm<sup>-1</sup> and two modes at ∼866 and 1572 cm<sup>-1</sup> before HBDI reaches the twisted intramolecular charge transfer (TICT) state on the ∼3 ps time scale. Remarkably, the wavelet transform analysis reveals a ∼500 fs delayed onset of the coupling peaks, in correlation with the emergence of an intermediate charge-separated state en route to the TICT state. This mechanism is corroborated by the altered coupling matrix for the HBDI Raman modes in the 50% (v/v) water-glycerol mixture, and a notable lengthening of the picosecond time constant. The real-time molecular "movie" of the general rotor-like HBDI isomerization reaction following photoexcitation represents a significant advance in comprehending the photochemical reaction pathways of the solvated GFP chromophore, therefore providing a crucial foundation to enable rational design of diverse nanomachines from efficient molecular rotors to bright fluorescent probes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Rozanov2018" class="col-sm-8"> <div class="title">MHC class I loaded ligands from breast cancer cell lines: A potential HLA-I-typed antigen collection</div> <div class="author"> Dmitri V. Rozanov, <em>Nikita D. Rozanov</em>, Kami E. Chiotti, Ashok Reddy, Phillip A. Wilmarth, Larry L. David, Seung W. Cha, Sunghee Woo, Pavel Pevzner, Vineet Bafna, Gregory G. Burrows, Juha K. Rantala, Trevor Levin, Pavana Anur, Katie Johnson-Camacho, Shaadi Tabatabaei, Daniel J. Munson, Tullia C. Bruno, Jill E. Slansky, John W. Kappler, Naoto Hirano, Sebastian Boegel, Bernard A. Fox, Colt Egelston, Diana L. Simons, Grecia Jimenez, Peter P. Lee, Joe W. Gray, and <a href="https://www.ohsu.edu/people/paul-t-spellman-phd" rel="external nofollow noopener" target="_blank">Paul T. Spellman</a> </div> <div class="periodical"> <em>Journal of Proteomics</em> 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.dx.doi.org/10.1016/j.jprot.2018.01.004" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="external nofollow noopener">Link</a> </div> <div class="abstract hidden"> <p>To build a catalog of peptides presented by breast cancer cells, we undertook systematic MHC class I immunoprecipitation followed by elution of MHC class I-loaded peptides in breast cancer cells. We determined the sequence of 3196 MHC class I ligands representing 1921 proteins from a panel of 20 breast cancer cell lines. After removing duplicate peptides, i.e., the same peptide eluted from more than one cell line, the total number of unique peptides was 2740. Of the unique peptides eluted, more than 1750 had been previously identified, and of these, sixteen have been shown to be immunogenic. Importantly, half of these immunogenic peptides were shared between different breast cancer cell lines. MHC class I binding probability was used to plot the distribution of the eluted peptides in accordance with the binding score for each breast cancer cell line. We also determined that the tested breast cancer cells presented 89 mutation-containing peptides and peptides derived from aberrantly translated genes, 7 of which were shared between four or two different cell lines. Overall, the high throughput identification of MHC class I-loaded peptides is an effective strategy for systematic characterization of cancer peptides, and could be employed for design of multi-peptide anticancer vaccines. Significance: By employing proteomic analyses of eluted peptides from breast cancer cells, the current study has built an initial HLA-I-typed antigen collection for breast cancer research. It was also determined that immunogenic epitopes can be identified using established cell lines and that shared immunogenic peptides can be found in different cancer types such as breast cancer and leukemia. Importantly, out of 3196 eluted peptides that included duplicate peptides in different cells 89 peptides either contained mutation in their sequence or were derived from aberrant translation suggesting that mutation-containing epitopes are on the order of 2–3% in breast cancer cells. Finally, our results suggest that interfering with MHC class I function is one of the mechanisms of how tumor cells escape immune system attack.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Oscar2016" class="col-sm-8"> <div class="title">Ultrafast intermolecular proton transfer to a proton scavenger in an organic solvent</div> <div class="author"> Breland G. Oscar, Weimin Liu, <em>Nikita D. Rozanov</em>, and <a href="https://chemistry.oregonstate.edu/Fang" rel="external nofollow noopener" target="_blank">Chong Fang</a> </div> <div class="periodical"> <em>Phys. Chem. Chem. Phys.</em> Sep 2016 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.dx.doi.org/10.1039/C6CP05692J" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="external nofollow noopener">Link</a> </div> <div class="abstract hidden"> <p>Proton transfer reactions are functionally important in numerous chemical and biological processes. To unravel proton scavengers in action with atomistic details, we studied excited-state proton transfer (ESPT) from photoacid pyranine to the weak base acetate in methanol using transient absorption and wavelength-tunable femtosecond stimulated Raman spectroscopy (FSRS). Proton transfer is inhibited in neat methanol, but coherent proton motions and the formation of a charge-separated state occur on the sub-picosecond (sub-ps) timescale, accompanied by chromophore solvation wherein the longitudinal relaxation time of methanol (∼9 ps) dominates. With acetate ions added, bimolecular diffusion-controlled ESPT from the photoacid to acetate occurs on the ∼30 ps timescale, followed by ∼600 ps diffusion-assisted charge separation and solvation in the methanol H-bonding network. Besides intensity dynamics, frequency redshift and blueshift of the transient ∼285 and 1525 cm<sup>−1</sup> modes track ESPT after 400 nm photoexcitation. Tunable FSRS exploits resonance Raman enhancement with optimal wavelengths, extends the detection window of excited-state vibrational modes to low frequency, and enables a deeper mechanistic understanding of the proton transfer reaction to proton scavengers in an organic solvent.</p> </div> </div> </div> </li> </ol> <h1>Senior Project/Thesis</h1> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Rozanov2018" class="col-sm-8"> <div class="title">Molecular Dynamics on REX-GECO1 Reveal Structural Features Governing Fluorescence</div> <div class="author"> <em>Nikita D. Rozanov</em> </div> <div class="periodical"> 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://ir.library.oregonstate.edu/concern/undergraduate_thesis_or_projects/2z10wx846" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://nikitaroz.github.io/assets/pdf/Senior-Thesis.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>Fluorescent proteins have emerged as an essential toolset for bioimaging, creating a demand for engineering proteins with new and improved fluorescent properties. In this thesis, I explore the atomistic structure of REX-GECO1, a newly engineered protein biosensor that has unique optical properties. Since this protein has no available crystal structure, understanding the relationship between its structure and properties is difficult. To overcome this challenge, I use molecular dynamics (MD) simulations to predict the protein’s structure and use this information to identify structural features that influence fluorescence. Moreover, I use the simulations to obtain thermodynamic information that provides further detail about the protein. These findings will be useful for understanding data obtained from ongoing ultrafast spectroscopic studies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Rozanov2019" class="col-sm-8"> <div class="title">AIChE Design Competition</div> <div class="author"> <em>Nikita D. Rozanov</em>, Max Morrow, Adam Marion, and Benjamin Avery</div> <div class="periodical"> 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://nikitaroz.github.io/assets/pdf/Senior-Design.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> <a href="https://nikitaroz.github.io/assets/pdf/AICHEs-and-Pains.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Poster</a> </div> <div class="abstract hidden"> <p>We, the AIChEs and Pains, completed the design for a facility capable of producing 1000 kg of a monoclonal antibody proteins annually. Per our calculations, our design will require a direct fixed capital cost of $38 million to complete construction of the plant. The annual operating cost for the plant will be $58 million. Competitor antibody manufacturers have a selling price point of around 5 million per kilogram of antibodies. Using this figure, the annual revenue for the plant will be around $4.9 billion. We believe further verification is required to determine the accuracy of this costing calculation. We recommend proceeding with a more detailed design after the costing calculations are verified a nd more accurately determined.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2023 Nikita Rozanov. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>